oidc-client-app-samples/android-client-app-sample/app/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179